src/databricks/labs/ucx/cli.py

```diff
@@ -48,17 +61,11 @@ def skip(schema: str, table: str | None = None):
         return None
     ws = WorkspaceClient()
     installation_manager = WorkspaceInstaller(ws)
-    logger.info("Fetching installation config.")
```

Comment:
don't refactor the functions, for now. also - use the proper InstallationManager (and not the WorkspaceInstaller)

https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/cli.py#L84-L87